src/Features/Core/Portable/EmbeddedLanguages/DateAndTime/DateAndTimeEmbeddedCompletionProvider.cs

```diff
@@ -76,8 +76,8 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
                 return;
 
             var stringToken = stringTokenOpt.Value;
-            if (position <= stringToken.SpanStart || position >= stringToken.Span.End)
-                return;
+            //if (position <= stringToken.SpanStart || position >= stringToken.Span.End)
+            //    return;
```

Comment:
I'm not sure what to do with this line. CompletionProvider receives position of '}' of format expression inside of string interpolation. I get previous token for analization, but SpanStart for the token is ```position - 1```. I've temporarily commented this line. @CyrusNajmabadi  Do you have any idea what the best way to handle it?

Reply:
i'm not really understanding the issue.  stringTokenOpt should be the appropriate string token for either case, unless the caller isn't passing the right thing.